tgui/packages/tgui/interfaces/Wizard_Spellbook.js

```diff
@@ -105,32 +105,54 @@ const Spell = (props, context) => {
   const { spell, category } = props;
 
   const [purchased, setPurchased] = useSharedState(context, spell+"p", false);
-  let spell_contents = []; // Non-associated list of: desc, cost, cooldown, vr_allowed
+  let spell_contents = []; // Non-associated list of: desc, cost, cooldown, vr_allowed, spell_img
   for (let spell_data in spellbook_contents[category][spell]) {
     spell_contents.push(spellbook_contents[category][spell][spell_data]);
   }
 
   return (
     <Stack.Item>
-      <Section>
+      <Section mt={0.5} mb={-1.5}>
         {(vr === 1) && (spell_contents[3] === 0) && (
-          <Dimmer>
+          <Dimmer mb={3} mt={-2}>
             <Box fontSize={1.5} backgroundColor={"#384e68"} p={2}>
               Spell unavailable in VR
             </Box>
           </Dimmer>
         )}
         <Section
-          title={spell}
-          buttons={
-            <Flex>
-              <Button
-                backgroundColor={"green"}
-                disabled={spell_slots < spell_contents[1] || purchased}
-                onClick={() => { setPurchased(true); act("buyspell", { spell: spell }); }}
-              >
-                {If_Purchased_Text(purchased, spell_contents[1], spell_slots)}
-              </Button>
+          title={
+            <Flex justify={"space-between"} align={"end"}>
+              <Flex.Item>
+                <Flex>
+                  <Flex.Item>
+                    {!!spell_contents[4] && (
+                      <Image
+                        pixelated
+                        mt={-2}
+                        height="38px"
+                        width="38px"
```

Comment:
Justification for this size? Is this the resolution of the icons? At a glance they seem to be 32x32.

Reply:
Its the size of the icons, 32x32 felt too little to notice the details of the icon and 38x38 felt just nice to see the details but not being too big.
Here is a comparison (The alignment for 32x32 is a bit wrong but that isnt the problem):
![image](https://github.com/goonstation/goonstation/assets/137294088/bc7f2730-13b1-4b82-b23d-873ab986bd5a)
![image](https://github.com/goonstation/goonstation/assets/137294088/a2766667-5522-47d3-8351-0f8188c264fb)

Reply:
I get that, but you're distorting a 32x32 image and it's having to make-up content for it (as the scale isn't an integer one). Note how in your example above the orb is slightly vertically stretched in the 38x38 version.

Reply:
Hmm, that's fair. I didn't think about how its actually distorting the image, I am fine with reverting it to 32x32 then.